src/routes.tsx

```diff
@@ -253,6 +256,16 @@ export const reactPageRoutes: ReactPageRoute[] = [
                       ? EmbarkStory.SwedenSwitcherSwedish
                       : EmbarkStory.SwedenSwitcherEnglish,
                 }
+              case 'home-accident-needer':
+                return {
+                  baseUrl: `/${locale}/new-member/home-accident-needer`,
+                  name: EmbarkStory.SwedenNeeder,
+                }
+              case 'home-accident-switcher':
+                return {
+                  baseUrl: `/${locale}/new-member/home-accident-switcher`,
+                  name: EmbarkStory.SwedenSwitcher,
+                }
```

Comment:
Is the plan to replace to other Swedish flows with the new one? If so, when would we be able to do that?

Reply:
Correct! Probably the last thing we do before going live :)